doozerlib/cli/release_gen_payload.py

```diff
@@ -447,6 +449,11 @@ def release_gen_payload(runtime: Runtime, is_name: str, is_namespace: str, organ
                 with output_path.joinpath(f"updated-tags-for.{imagestream_namespace}.{imagestream_name}{'-partial' if incomplete_payload_update else ''}.yaml").open("w+", encoding="utf-8") as out_file:
                     istream_spec = PayloadGenerator.build_payload_imagestream(imagestream_name, imagestream_namespace, istags, assembly_issues)
                     yaml.safe_dump(istream_spec, out_file, indent=2, default_flow_style=False)
+                    if publish:
+                        publish_name = f"{runtime.get_minor_version()}.0-{runtime.assembly}"
+                        publish_reponame = imagestream_namespace.replace("ocp", "release")
```

Comment:
This seems to be dangerous to do a string replacement because you are making an assumption that there is one but only one `ocp` substring in the imagestream_namespace. If it changes, this will probably overwrite things unexpectedly. I believe to be explicitly configured either by command line arguments or config files.